src/aiida_quantumespresso/calculations/matdyn.py

```diff
@@ -55,11 +62,20 @@ def _validate_inputs(value, _):
         if 'parameters' in value:
             parameters = value['parameters'].get_dict()
         else:
-            parameters = {}
+            parameters = {'INPUT': {}}
```

Comment:
Note that if the `parameters` contains an empty dict, this will now fail. Rather initialize empty dict and ensure the `INPUT` sub dict is there in both bases
```suggestion
            parameters = {}

        parameters.setdefault('INPUT', {})
```

Reply:
That said, if the user provides an _empty_ `Dict` node for `parameters`, they might as well have not provided the input (cleaner provenance). So perhaps we should validate that the `INPUT` key is in the `parameters` input instead?

Reply:
See f659d2c53922d0340fb3c9f7905ecb8711a52974